.projenrc.js

```diff
@@ -145,7 +145,10 @@ fetchAssemblies.exec(`node scripts/fetch-assemblies.js`);
 project.gitignore.exclude("public/packages");
 
 // Proxy requests to awscdk.io for local testing
-project.package.addField("proxy", "https://awscdk.io");
+project.package.addField(
+  "proxy",
+  "https://construct-hub-testing.dev-tools.aws.dev/"
```

Comment:
This is kind of funky. Any other ideas?

Reply:
Could we read this value from env or some sort of configuration file?

Reply:
Mmm maybe config file we ignore. let me try.

Reply:
@gabewomble This won't work since eventually it gets written to `package.json` - so we will always run the risk of committing it.

Reply:
Actually, if this is just going to hit the same domain that it's deployed to, do we need a proxy?

Reply:
> Forgot about local dev... the env I use the most

Ha :)

Anyway - its ok, we'll leave it like this.